src/test/java/io/jenkins/plugins/analysis/core/charts/SeverityPieChartTest.java

```diff
@@ -0,0 +1,96 @@
+package io.jenkins.plugins.analysis.core.charts;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import edu.hm.hafner.analysis.Report;
+import edu.hm.hafner.analysis.Severity;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+/**
+ * Tests the class {@link SeverityPieChart}.
+ *
+ * @author Matthias Herpers
+ */
+class SeverityPieChartTest {
+
+    /**
+     *  Verifies that a SeverityPieChart with empty report shows only predefined severities without Severity.ERROR.
+     */
+    @Test
+    void buildTestWithEmptyReport() {
```

Comment:
Better use a more meaningful method name starting with should eg. ShouldCreateChartFromEmptyReport (cf. [https://github.com/uhafner/codingstyle/blob/master/doc/Testen.md](url)) (applies also to the other method names)